foreman/foreman.spec

```diff
@@ -715,6 +715,7 @@ install -Dpm0644 %{SOURCE8} %{buildroot}%{_sysconfdir}/pki/rpm-gpg/RPM-GPG-KEY-f
 cp -p Gemfile.in %{buildroot}%{_datadir}/%{name}/Gemfile.in
 cp -p -r app bin bundler.d config config.ru extras lib locale Rakefile script %{buildroot}%{_datadir}/%{name}
 rm -rf %{buildroot}%{_datadir}/%{name}/extras/{jumpstart,spec}
+rm -rf %{buildroot}%{_datadir}/%{name}/script/%{name}-tail.d/{common,redhat,debian}
```

Comment:
This doesn't fix the problem I described, the bug is liable to recur if any directories are added.

Delete _all_ directories beneath `%{name}-tail.d`, there shouldn't be any. The `find .. -type d` command may help.